src/govuk/components/accordion/template.njk

```diff
@@ -1,8 +1,15 @@
 {% set id = params.id %}
 {% set headingLevel = params.headingLevel if params.headingLevel else 2 %}
 
-<div class="govuk-accordion {%- if params.classes %} {{ params.classes }}{% endif -%}" data-module="govuk-accordion" id="{{ id }}"
-{%- for attribute, value in params.attributes %} {{attribute}}="{{value}}"{% endfor %}>
+<div
+  class="govuk-accordion {%- if params.classes %} {{ params.classes }}{% endif -%}"
```

Comment:
I'm also not 100% sure about splitting attributes over multiple lines, in terms of the readability of the HTML that's outputted. (It's fine to split the Nunjucks wrapped attributes out, as the whitespace control puts them back on one line anyway)

We use a prettifier on the HTML examples in the Design System, but I don't think it removes extra new lines like these ones. Worth checking with a pre-release to see what it looks like?